examples/hybrid/driver.jl

```diff
@@ -394,13 +394,18 @@ if parsed_args["regression_test"]
 end
 
 if parsed_args["check_conservation"]
-    @test sum(sol.u[1].c.ρ) ≈ sum(sol.u[end].c.ρ) rtol = 20 * eps(FT)
-    @test sum(sol.u[1].c.ρe_tot) +
-          (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][]) ≈
-          sum(sol.u[end].c.ρe_tot) rtol = 20 * eps(FT)
+    @test abs(sum(sol.u[1].c.ρ) - sum(sol.u[end].c.ρ)) / sum(sol.u[1].c.ρ) ≈
+          FT(0) atol = 4 * eps(FT)
+    @test abs(
+        sum(sol.u[1].c.ρe_tot) +
+        (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][]) -
+        sum(sol.u[end].c.ρe_tot),
+    ) / (
+        sum(sol.u[1].c.ρe_tot) +
+        (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][])
+    ) ≈ FT(0) atol = 2 eps(FT)
```

Comment:
@simonbyrne here is where I found that keeping the test with `atol` passes (and with much smaller tolerances than the original version), but not with `rtol`

Reply:
So `isapprox(x, y; atol)` will check absolute error
```
norm(x - y) < atol
```
and `isapprox(x, y; rtol)` does relative error
```
norm(x - y) < rtol * max(norm(x), norm(y))
```

What you're doing is trying manually computing the relative error and comparing it to zero: i.e. you're doing
```
isappox((x - y)/x, 0)
```
where `x = sum(sol.u[1].c.ρe_tot) + (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][])` and `y = sum(sol.u[end].c.ρe_tot)`, and so the error has already been "relativized"

Probably what you want is
```suggestion
    @test sum(sol.u[1].c.ρ) ≈ sum(sol.u[end].c.ρ)) rtol = 4 * eps(FT)
    @test sum(sol.u[1].c.ρe_tot) + (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][]) ≈
        sum(sol.u[end].c.ρe_tot) rtol = 2 eps(FT)
```